packages/dds/tree/src/test/feature-libraries/modular-schema/comparison.spec.ts

```diff
@@ -200,6 +209,123 @@ describe("Schema Comparison", () => {
 		);
 	});
 
+	/**
+	 * This test suite aims to test the ordering relationship between the different repositories
+	 * (i.e different combinations of field schemas), it will not primarily focus on the comparison
+	 * between field schemas themselves, the tests for `allowsFieldSuperset` and `allowsTreeSuperset`
+	 * have done most of the heavy-lifting.
+	 *
+	 * The primary objective behind designing this test suite is to ensure backward compatibility
```

Comment:
I don't think this paragraph is necessary (or true, really--this function helps support our plans for backward compatibility, but the unit tests here are just making sure it's correct). If you want to go with something like:

"This helps provide some coverage for our schema evolution story, since repo compatibility influences the types of schema changes we allow."

that's fair :)